src/olympia/abuse/models.py

```diff
@@ -96,7 +96,7 @@ class AbuseReport(ModelBase):
         ('ABOUT_DEBUGGING', 2, 'Add-ons Debugging'),
         ('ABOUT_PREFERENCES', 3, 'Preferences'),
         ('AMO', 4, 'AMO'),
-        ('APP:PROFILE', 5, 'app:profile'),
+        ('APP_PROFILE', 5, 'App Profile'),
```

Comment:
checking this doesn't need a DB migration for any existing records with the value `APP:PROFILE`

Reply:
We just store the integer value (`5`) so no (and it's a recent introduction anyway, no clients except maybe Firefox Nightly would have sent this parameter, and if they did they would have sent `app_profile` which would not have worked before this patch)